### PR TITLE
feat: web and docs refactoring

### PR DESCRIPTION
**Pull request by Not-Sarthak**
- Docs
  - first get request update
  - first post request update
  - get token update
- Website
   - 3d sphere
   - infinite carousel
   - minor bug fixes
- Build scripts

### CODE REVIEW

**Comment by Not-Sarthak** [comment]
- Website
  - Loader

File: apps/docs/pages/developer/get-request/first-get-request.mdx [hunk 1]
```diff
@@ -29,10 +29,10 @@ touch 0rbit-Get-Request.lua
 local json = require("json")
 
 _0RBIT = "BaMK1dfayo75s3q1ow6AO64UDpD9SEFbeE8xYrY2fyQ"
-_0RBT_TOKEN = "BUhZLMwQ6yZHguLtJYA5lLUa9LQzLXMXRfaq9FVcPJc"
+_0RBT_POINTS = "BUhZLMwQ6yZHguLtJYA5lLUa9LQzLXMXRfaq9FVcPJc"
 
 FEE_AMOUNT = "1000000000000" -- 1 $0RBT
-BASE_URL = "https://dummyjson.com/products"
+BASE_URL = "https://api.diadata.org/v1/assetQuotation/Arweave/0x0000000000000000000000000000000000000000"
```
Comment (megabyte0x) [inline apps/docs/pages/developer/get-request/first-get-request.mdx:35]: any reason for not using coingecko api?

**Comment by megabyte0x** [comment]
@Not-Sarthak The logo of X (Twitter) on the footer in docs is the old one; kindly correct that, too.

File: apps/docs/pages/developer/get-request/first-get-request.mdx (hunk 1 above)
Comment (Not-Sarthak) [reply apps/docs/pages/developer/get-request/first-get-request.mdx:35]: yes, it had some issues with the free api and hence used this instead.
https://docs.coingecko.com/reference/simple-price

**Review by megabyte0x — CHANGES_REQUESTED** [review]

File: apps/web/app/layout.tsx [hunk 2]
```diff
@@ -84,7 +84,7 @@ export default function RootLayout({
         className={`${raleway.className} min-h-[100vh] ${brandLightBg} flex flex-col items-center`}
       >
         {" "}
-        <Nav />
+        {/* <Nav /> */}
```
Comment (megabyte0x) [inline apps/web/app/layout.tsx:87]: if it's not required, then remove it.

File: apps/web/app/_components/_landingContainer/Features.tsx [hunk 3]
```diff
@@ -27,7 +28,8 @@ const Features = () => {
           lg:w-[240px] lg:h-[240px] 
           md:w-[180px] md:h-[180px] 
           w-[180px] h-[180px] rotating"
```
Comment (megabyte0x) [inline apps/web/app/_components/_landingContainer/Features.tsx:30]: Could you please remove the unwanted comments?

**Review by megabyte0x — CHANGES_REQUESTED** [review]

**Comment by megabyte0x** [comment]
https://github.com/user-attachments/assets/909df38b-d11b-46d5-9b20-9456a95a9ae0

@0xLPircy  The mobile screen is moving horizontally now

**Pull request merged**